ASWF/meetings/tsc/2020-09-28.md

```diff
@@ -0,0 +1,222 @@
+      this to high priority issues. This issue is to create OFX bindings for 
+      OCIO v2. High priority since it would increase test coverage with so many 
+      supporting apps. Could get more practitioners and developers to download 
+      and test those.
+    - Michael: OFX is supported by Nuke; Curious if it would create conflict 
+      with existing OCIO v1 binary in that env.
+
+* Open PRs: 
+    - 1147 (optimization):
+        - Doug: Updates config default version and optimization improvements. 
+          When create new config, it is now v2 by default instead of v1. Other 
+          thing in PR enhances optimization flags. Now ability to control types 
+          of optimization. Two new flags: one simplifies ops down to component 
+          parts; CDL without power becomes just matrix if possible, etc. 
+          Benefit is that can be combined with another matrix op. Another flag 
+          controls whether SSE log and power are used. Does much faster version 
+          of these functions but not 32-bit exact - 16-bit half precision. 
+          Useful to turn it on and off, and off for unit tests since we’re 
+          comparing with higher precision results.
+        - Bernard: Fast log is also used for baking.
+        - Mark: Why is SSE path not as precise? Because of writing own exponent 
+          functions etc?
+        - Patrick: Yes, wrote own implementation. Benefit in performance is so 
+          important that both approaches are supported.
+        - Doug: Uses fewer iterations than needed for full float 32-bit results.
+        - Mark: Had similar challenge getting this working in CTL expression 
+          code. Finding concise method of operations challenging. Since we 
+          already have concept of optimizations sounds fine. Is this default?
+        - Doug: Yes, both are turned on by default, in OPTIMIZATION_VERY_GOOD.
+        - Patrick: optimization is based on bits, so can disable any of the 
+          items. Fine control of optimization. There was a question from Zach 
+          Lewis about v1. You can still make v1 config by setting version.
+    - 1157 (file transform interpolation):
+        - Doug: Prompted by issue (1121) from Remi. File transform will now use 
+          DEFAULT interpolation as default, instead of UNKNOWN, which prevents 
+          exceptions being thrown in various places. Also cleaned up some file 
+          transform interpolation. If you specify an interpolation that can't 
+          be used for file, will log warning. For CLF, where interpolation can 
+          be specified in file, if an op doesn't specify interpolation, it will 
+          try to use the FileTransform interpolation specified. Will ignore 
+          file transform if specified in file. Also addresses issue with 
+          caching. If multiple transforms used specify different 
+          interpolations, that will now work.
+        - Bernard: Also added CAT transform function.
```

Comment:
?
Are we talking Chromatic Adaptation Transforms in the context of File Transform interpolation?

Reply:
There is nothing involving chromatic adaptation in this PR.  I think Bernard's comment was something else.  Suggest:

Bernard: Also added a function to get the type of a transform (CDL, Matrix, etc.).

Reply:
Corrected this based on Doug's suggestion